src/main/scala/cromwell/engine/SingleWorkflowRunner.scala

```diff
@@ -0,0 +1,45 @@
+package cromwell.engine
+
+import java.io.File
+
+import cromwell.binding.FullyQualifiedName
+import cromwell.binding.types._
+import cromwell.binding.values.WdlValue
+import cromwell.engine.WorkflowManagerActor._
+import cromwell.server.WorkflowManagerSystem
+import cromwell.util.{ActorUtil, FileUtil}
+import spray.json._
+import scala.concurrent.Future
+import scala.util.{Try, Failure}
+
+class SingleWorkflowRunner extends WorkflowManagerSystem {
+  def run(wdlFile: File, inputs: File): Try[Map[FullyQualifiedName, WdlValue]] = {
+    Try(FileUtil.slurp(inputs).parseJson) map {
+      case JsObject(rawInputs) =>
+        val workflowId = ActorUtil.messageAndWait(SubmitWorkflow(FileUtil.slurp(wdlFile), rawInputs), _.mapTo[WorkflowId])(workflowManagerActor)
+        println(s"Workflow ID: $workflowId")
+
+        /* TODO: horrible, terrible hack to wait until a workflow is succeeded */
```

Comment:
I have some code to clean this up but it needs to wait for my previous "actor store" PR to be merged.